docs/register-phase-1-stake.md

```diff
@@ -0,0 +1,154 @@
+  that were used when you created your Bitcoin stake. This will be used in the 
+  registration command later on.
+- A running instance of the `stakerd` daemon. Please follow the `stakerd` 
+  [setup guide](../README.md#3-btc-staker-installation) to complete the setup. 
+  This is also inclusive of creating a Babylon keyring with funds step as the 
+  `stakerd` daemon requires a funded keyring to pay for the transactions.
+
+## Setup and configuration
+
+> **⚡ Note**: While the `stakerd` daemon is already running, it needs access to 
+> the Bitcoin wallet containing your staking transaction's private key. This 
+> setup ensures the daemon can interact with the correct Bitcoin wallet.
+
+To get started, you will need to create a configuration file. You can do this 
+by running the `dump-config` command. This will create a `config.json` file 
+in your current working directory that you'll customize with your specific 
+values:
+
+```bash
+stakercli admin dump-config
+```
+
+The configuration file below contains default settings and placeholders 
+for your Bitcoin and Babylon values. Please replace the placeholders with 
+your specific information:
+
+```toml
+[walletconfig]
+WalletName = "btc-staker"  # Replace with your Bitcoin wallet name
+WalletPass = "your-wallet-password"  # Replace with your wallet password
+
+[walletrpcconfig]
+Host = "127.0.0.1:19001"  # Bitcoin Wallet RPC endpoint
+User = "your-rpc-user"    # Replace with your Bitcoin RPC username
+Pass = "your-rpc-pass"    # Replace with your Bitcoin RPC password
+
+[chain]
+Network = "signet"  # Bitcoin network (signet for testnet)
+```
+
+Finally, start the `btcstaker` daemon, which will manage your connections to both 
+Babylon and Bitcoin nodes, monitor transactions, and handle the staking process.
+Add the location for the above created `config.json` file to the command:
+
+```shell
+stakerd --configfile <config-file-path> admin start
+```
+
+## Register your stake
+
+After configuring and starting the `btcstaker` daemon, you'll need the 
+`global_parameters.json` file (created during prerequisites) for the next steps.
+
+A few notes about the `global-parameters.json` file:
+1. It is used to retrieve covenant keys and quorum from the parameters version 
+   you staked in.
+2. It can also be used to parse your Bitcoin staking transaction to extract 
+   important information from the `OP_RETURN` data, which contains:
+   - The staking period you chose
+   - The finality provider you chose
+
+> **⚡ Note**: While these parameters can be provided manually, the global 
+> parameters file provides an easier user experience. This guide is only for 
+> valid, confirmed Bitcoin staking transactions with valid `OP_RETURN` data. If 
+> your transaction has malformed `OP_RETURN` data or isn't a confirmed staking 
+> transaction, to check your eligibility please refer to the 
```

Comment:
```suggestion
> transaction (because it is a new one or very recently submitted), it cannot be registered.
> To check your registration eligibility please refer to the 
```